centralized-pre-commit-conf/conda/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: centralized-pre-commit-conf
+  version: 0.6.1
+
+source:
+  git_url: https://github.com/Pierre-Sassoulas/centralized-pre-commit-conf
+  git_rev: v0.6.1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - pre-commit-conf = centralized_pre_commit_conf.main:run
+
+test:
+  requires:
+    - pytest-cov
+    - pytest-vcr
+    - python-coveralls
+    - coverage
+    - coveralls
+    - pylint =2.15.4
+  imports:
+    - centralized_pre_commit_conf
+
```

Comment:
Add 

commands:
  - pre-commit-conf -h
  - tests

Reply:
I did a slightly different change that works too and is more in line with our usual conda recipes:
```
  commands:
    - pytest
```